src/vma/lwip/tcp_out.c

```diff
@@ -921,24 +922,29 @@ tcp_split_segment(struct tcp_pcb *pcb, struct tcp_seg *seg, u32_t wnd)
     /* Update number of buffer to be send */
     pcb->snd_queuelen++;
   }
-  else if (seg->p->next) {//Segment with more than one pbuffer, split segment pbuff chain
+  else if (seg->p->next) {/*Segment with more than one pbuffer, split segment pbuff chain. At least one pBuffer will be sent*/
```

Comment:
1. suggested updated comment
```
/* Segment with more than one pbuffer and seg->p->len <= lentosend
       split segment pbuff chain. At least one pBuffer will be sent */
```
2. Keep a space between * and beginning/ending of text (go all over the file).  /\*Segment ==> /\* Segment